app/controllers/organisations_controller.rb

```diff
@@ -6,7 +6,9 @@ class OrganisationsController < ApplicationController
   before_action :find_resource, except: %i[index new create]
   before_action :authenticate_scope!, except: [:index]
   before_action :session_filters, if: -> { current_user.support? || current_user.organisation.has_managing_agents? }, only: %i[lettings_logs sales_logs email_lettings_csv download_lettings_csv email_sales_csv download_sales_csv]
+  before_action :session_filters, only: %i[users]
```

Comment:
Assuming this is intentionally not included with the line above

Reply:
Yeah, it's cause the conditions for running this are different on different paths. Didn't want to clutter the above line with too much logic either